osu.Game/Overlays/News/NewsCard.cs

```diff
@@ -161,5 +163,10 @@ private void load(OverlayColourProvider colourProvider)
 
             DateTimeOffset IHasCustomTooltip<DateTimeOffset>.TooltipContent => date;
         }
+
+        public MenuItem[] ContextMenuItems => new MenuItem[]
+        {
+            new OsuMenuItem("View news in browser", MenuItemType.Highlighted, Action),
```

Comment:
Not localised since "in browser" is temporary, and removing it will cause confusion.

Reply:
I don't know how temporary it is. News posts heavily use inline HTML, especially for things like audio or YouTube embeds, and I'm not sure we're ever supporting that.

Probably fine for now either way, but just food for thought I guess.